lang/en_US.json

```diff
@@ -634,7 +634,7 @@
     "Mastered by same players": "Mastered by same players",
     "<1>{{achievementTitle}}</1> from <2>{{gameTitle}}</2>": "<1>{{achievementTitle}}</1> from <2>{{gameTitle}}</2>",
     "New usernames must match.": "New usernames must match.",
-    "You can only request a new username once every 30 days, even if your new username is not approved. Are you sure you want to do this?": "You can only request a new username once every 30 days, even if your new username is not approved. Are you sure you want to do this?",
+    "Username change requests have cooldown periods: 90 days after an approval, or 30 days after a denial. Are you sure you want to request a change?": "Username change requests have cooldown periods: 90 days after an approval, or 30 days after a denial. Are you sure you want to request a change?",
```

Comment:
The implementation seems to require 90 days for both approvals and denials.

Reply:
Fixed in latest with an update to the policy.